src/execution_plan/ops/op_merge_create.c

```diff
@@ -34,7 +34,7 @@ static void _IncrementalHashEntity(XXH64_state_t *state, const char **labels,
 		res = XXH64_update(state, &_set->attr_count, sizeof(_set->attr_count));
 		ASSERT(res != XXH_ERROR);
 
-		for(int i = 0; i < _set->attr_count; i++) {
+		for (uint16_t i = 0; i < _set->attr_count; ++i) {
```

Comment:
if there's no good reason to use prefix addition please switch to suffix addition `I++`